drivers/timer/stm32_lptim_timer.c

```diff
@@ -394,14 +404,29 @@ static int sys_clock_driver_init(void)
 		 * Time base = (4s * freq) - 1
 		 */
 		lptim_clock_freq = lptim_clock_freq / 2;
+		lptim_clock_presc = lptim_clock_presc * 2;
```

Comment:
That would deserve a comment

Reply:
this section is removed after PR https://github.com/zephyrproject-rtos/zephyr/pull/65683